src/xrpld/app/tx/detail/OfferStream.cpp

```diff
@@ -339,7 +340,9 @@ TOfferStreamBase<TIn, TOut>::step()
                                 std::is_same_v<TOut, XRPAmount>))
                     return shouldRmSmallIncreasedQOffer<IOUAmount, IOUAmount>();
             }
-            assert(0);  // xrp/xrp offer!?! should never happen
+            UNREACHABLE(
+                "rippls::TOfferStreamBase::step::rmSmallIncreasedQOffer : XRP "
+                "vs XRP offer");  // xrp/xrp offer!?! should never happen
```

Comment:
```suggestion
                "vs XRP offer");
```

The extra comment is now superfluous.